RELEASES.md

```diff
@@ -90,43 +85,53 @@ whichever is longer. Additionally, releases may have an extended security suppor
-| [1.4](https://github.com/containerd/containerd/releases/tag/v1.4.13) | End of Life   | August 17, 2020    | March 3, 2022                                           |
-| [1.5](https://github.com/containerd/containerd/releases/tag/v1.5.18) | End of Life   | May 3, 2021        | February 28, 2023                                       |
-| [1.6](https://github.com/containerd/containerd/releases/tag/v1.6.36) | LTS           | February 15, 2022  | next LTS + 6 months                                     |
-| [1.7](https://github.com/containerd/containerd/releases/tag/v1.7.23) | Active        | March 10, 2023     | active(May 5, 2025), extended(EOL of 1.6)               |
-| [2.0](https://github.com/containerd/containerd/releases/tag/v2.0.0)  | Active        | November 5, 2024   | max(November 5, 2025 or release of 2.1 + 6 months)      |
-| [2.1](https://github.com/containerd/containerd/milestone/48)         | Next          | TBD                | TBD                                                     |
-
-> **_NOTE_** containerd v1.7 will end of life at the same time as v1.6 LTS. Due to
-> [Minimal Version Selection](https://go.dev/ref/mod#minimal-version-selection) used
-> by Go modules, 1.7 must be supported until EOL of all 1.x releases. Once 1.7 is in
-> extended support, it will continue to accept security patches in addition to client
-> changes relevant for package importers using the 1.6 LTS daemon.
+Long term stable (_LTS_) releases are owned by at least two maintainers for at least two
+years after their initial _minor_ (x.y.0) release. The maintainers of the _LTS_ branch may commit to
+a longer period or extend the support period as needed. These branches will accept bug reports and
+backports until the end of life date. They may also accept a wider range of patches than non-_LTS_
+releases to support the longer term maintainability of the branch, including library dependency,
+toolchain (including Go) and other version updates which are needed to ensure each release is built
+with fully supported dependencies. Feature backports are up to the discretion of the maintainers who
+own the branch but should be rejected by default. There is no defined limit to the number of _LTS_
+branches and any branch may become an _LTS_ branch after its initial release. There is no guarantee
+that a new _LTS_ branch will be designated before existing _LTS_ branches reach their end of life.
+
+### Release Owners
+
+Every release shall be assigned owners when entering into the beta stage of the release. The initial
+release owners will act as the release captains and will be responsible for creating the releases and
+ensuring the release is on time. Once the release is in rc, the release owners should be part of any
+discussion around merging impactful or risky changes. Every release should have two owners who
+are both active maintainers and one of which has been involved in at least two prior releases.
```

Comment:
What are the "involved in" requirements?  Does that just mean "a maintainer during the Active state of that release" or acting in some other capacity?

Reply:
I'll update to has been a release owner to make it more explicit. The idea is that it would allow non-committers to be involved with releases. We can leave it up to us how we would define release owners for prior releases, but I plan to be involved in the next two scheduled releases to help transition the process anyway. I can update to "at least" two if that will help train more folks if necessary.